Server/AIServer/ServerDlg.cpp

```diff
@@ -1153,8 +1166,11 @@ BOOL CServerDlg::MapFileLoad()
 			CFile file;
 			if (!file.Open(szFullPath, CFile::modeRead))
 			{
-				errormsg.Format(_T("파일 Open 실패 - %s\n"), szFullPath.GetString());
-				AfxMessageBox(errormsg);
+				std::string error = std::format("MapFileLoad: Failed to open file: {}", mapPath.generic_string());
```

Comment:
Okay, because this one's a path it gets messy. But here's what you can do.
Make the base error a wide string. Fetch the path as a native_string() (which is std::wstring) and convert the entire error with WideToUtf8() from shared/StringConversion.h

Then pass the utf-8 error to spdlog, and pass the wide error to AfxMessageBox.